lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -127,7 +127,7 @@ export class PopupClient extends StandardInteractionClient {
             }
 
             if (e instanceof AuthError) {
-                e.setCorrelationId(this.correlationId);
+                (e as AuthError).setCorrelationId(this.correlationId);
```

Comment:
Is TypeScript really complaining about this? Isn't this working today?

Reply:
Depends on local linter I assume